proto/teleport/lib/teleterm/v1/service.proto

```diff
@@ -264,15 +264,15 @@ message LoginPasswordlessRequest {
 }
 
 message FileTransferRequest {
-  string cluster_uri = 1;
+  // login is the SSH username to use for the transfer.
   string login = 2;
-  string hostname = 3;
-  // source path of the transferred file
+  // source path of the transferred file.
   string source = 4;
-  // destination path of the transferred file
+  // destination path of the transferred file.
   string destination = 5;
-  // indicates whether the file is uploaded/downloaded
+  // direction indicates whether the file is uploaded/downloaded.
   FileTransferDirection direction = 6;
+  string server_uri = 7;
```

Comment:
IMO we could just have `string server_uri = 1` as Connect is the only client of this RPC.

Reply:
Yeah I know, I keep going back and forth on this. But then it'd also make sense to adjust the numbers on other fields (since we're removing two and adding one) and that generates a lot of changes so I'm just like… maybe we should strive to follow best practices regardless of the fact that Connect is the only client.

Reply:
I don't have a strong opinion, we can do this 👍